### PR TITLE
[src] updated the Workload section

### PR DESCRIPTION
**Pull request by konstantin-lysenko-netapp**

### CODE REVIEW

File: api/services/ocean/cd/paths/workloadActiveOperations.yaml [hunk 1]
```diff
@@ -1,44 +1,12 @@
 summary: Ocean CD
 get:
-  summary: "Ocean CD Describe Workload's Active Operations"
-  description: >
-    Describe What Operation Are Active For Specific Workload.
-  operationId: "OceanCDCWorkloadActiveOperations"
+  summary: "Get Ocean CD Workload's Active Operations"
```
Comment (caduri) [inline api/services/ocean/cd/paths/workloadActiveOperations.yaml:3]: it should be active operation
not operations

File: api/services/ocean/cd/paths/workloadActiveOperations.yaml [hunk 2]
```diff
@@ -1,44 +1,12 @@
...
+  description: Get the ongoing operations being applied to the workload.
```
Comment (caduri) [inline api/services/ocean/cd/paths/workloadActiveOperations.yaml:4]: it should be operation

File: api/services/ocean/cd/paths/workloadActiveOperations.yaml [hunk 3]
```diff
@@ -1,44 +1,12 @@
...
+  operationId: "OceanCDWorkloadActiveOperations"
```
Comment (caduri) [inline api/services/ocean/cd/paths/workloadActiveOperations.yaml:5]: it should be operation

File: api/services/ocean/cd/paths/workloadActiveOperations.yaml [hunk 4]
```diff
@@ -1,44 +1,12 @@
...
-      example: SpotDeployment
-      description: >
-        Workload's k8s Kind, currently only SpotDeployment is supported
-      schema:
-        type: string
-      required: true
+    $ref: "../schemas/oceanCDWorkloadRequest.yaml"
   responses:
     200:
       $ref: "../responses/oceanCDWorkloadActiveOperationsResponse.yaml"
```
Comment (caduri) [inline api/services/ocean/cd/paths/workloadActiveOperations.yaml:12]: it should be active operation

File: api/services/ocean/cd/schemas/oceanCDWorkloadActiveOperations.yaml [hunk 5]
```diff
@@ -1,19 +1,24 @@
 type: object
-required:
-  - workload
+title: Ocean CD Workload Active Operations
```
Comment (caduri) [inline api/services/ocean/cd/schemas/oceanCDWorkloadActiveOperations.yaml:2]: it should be active operation

File: api/services/ocean/cd/schemas/oceanCDWorkloadActiveOperations.yaml [hunk 6]
```diff
@@ -1,19 +1,24 @@
...
+description: Indicates the ongoing operations being applied to the workload
```
Comment (caduri) [inline api/services/ocean/cd/schemas/oceanCDWorkloadActiveOperations.yaml:3]: it should be operation

File: api/services/ocean/cd/schemas/oceanCDWorkloadActiveOperations.yaml [hunk 7]
```diff
@@ -1,19 +1,24 @@
...
+    example: retry
   operationStatus:
     type: string
+    description: The status of the operation
+    example: finished
   createdAt:
     type: string
-    format: date-time
-    example: "2018-11-05T12:55:50.000+0000"
+    description: The time when the operation was created
```
Comment (caduri) [inline api/services/ocean/cd/schemas/oceanCDWorkloadActiveOperations.yaml:19]: why did you remove the format?

File: api/services/ocean/cd/schemas/oceanCDWorkload.yaml [hunk 8]
```diff
@@ -1,44 +1,40 @@
...
+    description: The number of the ready replicas
+    example: 2
   updatedAt:
     type: string
-    format: date-time
-    example: "2018-11-05T12:55:50.000+0000"
-    description: >
-      The timestamp of latest update
\ No newline at end of file
+    description: The last time when the workload was changed
```
Comment (caduri) [inline api/services/ocean/cd/schemas/oceanCDWorkload.yaml:39]: why did you remove the format?

File: api/services/ocean/cd/schemas/oceanCDWorkloadActiveOperations.yaml [hunk 9]
```diff
@@ -1,19 +1,24 @@
...
+    example: "2018-11-05T12:55:50.000Z"
   updatedAt:
     type: string
-    format: date-time
-    example: "2018-11-05T12:55:50.000+0000"
\ No newline at end of file
+    description: The last time when the operation was changed
```
Comment (caduri) [inline api/services/ocean/cd/schemas/oceanCDWorkloadActiveOperations.yaml:23]: why did you remove the format?

File: api/services/ocean/cd/schemas/oceanCDWorkloadRevision.yaml [hunk 10]
```diff
@@ -1,18 +1,24 @@
...
   rolloutId:
     type: string
-    example: "rol-67c2d5c2d384"
+    description: The identifier of the Ocean CD rollout
+    example: rol-67c2d5c2d384
   date:
     type: string
-    format: date-time
-    example: "2018-11-05T12:55:50.000+0000"
+    description: The time when the revision was defined
```
Comment (caduri) [inline api/services/ocean/cd/schemas/oceanCDWorkloadRevision.yaml:15]: why did you remove the format?

**Review by caduri — CHANGES_REQUESTED** [review]

File: api/services/ocean/cd/schemas/oceanCDWorkloadActiveOperations.yaml (hunk 7 above)
Comment (konstantin-lysenko-netapp) [reply api/services/ocean/cd/schemas/oceanCDWorkloadActiveOperations.yaml:19]: I'm not sure the field is informative. The format isn't the same as displayed in this field. Also, as we have an example, we don't need it, or let's add smth like `yyyy-mm-ddTHH:mm:ss.fffZ`

**Review by caduri — APPROVED** [review]
